src/screens/OtherProfile/index.js

```diff
@@ -32,9 +32,10 @@ import RenderItem from '../ProfileScreen/elements/RenderItem';
 import ReportUser from '../../components/Blocking/ReportUser';
 import ShareUtils from '../../utils/share';
 import SpecificIssue from '../../components/Blocking/SpecificIssue';
-import TextAreaChat from '../../components/TextAreaChat';
+import StorageUtils from '../../utils/storage';
 import dimen from '../../utils/dimen';
-import useSaveAnonChatHook from '../../database/hooks/useSaveAnonChatHook';
+import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import useCreateChat from '../../hooks/screen/useCreateChat';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useCoreFeed` and `useCreateChat` hooks is added but `TextAreaChat` and `useSaveAnonChatHook` are removed. Make sure these changes do not break any existing functionality.
```diff
- import TextAreaChat from '../../components/TextAreaChat';
- import useSaveAnonChatHook from '../../database/hooks/useSaveAnonChatHook';
+ import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+ import useCreateChat from '../../hooks/screen/useCreateChat';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->